azure-pipelines.yml

```diff
@@ -75,6 +87,9 @@ jobs:
       name: macOS
       vmImage: xcode9-macos10.13
       matrix:
+        py_3.5_64:
+          MB_PYTHON_VERSION: "3.5"
+          NP_BUILD_DEP: "numpy==1.13.3"
```

Comment:
Why note `1.12.1` as for Windows?

Reply:
They were different for py36, I just used the same as py36 for each platform.

Reply:
yes for macos, but not for windows. And we don't force the version for linux. I don't know why